tap_gorgias/streams.py

```diff
@@ -255,10 +259,80 @@ def sync(self, state, config):
             yield (self.stream, event)
         self.update_bookmark(state, max_synced_thru)
 
+class VoiceCallEvents(CursorStream):
+    name = 'voice_call_events'
+    replication_method = 'INCREMENTAL'
+    key_properties = ['id']
+    replication_key = 'created_datetime'
+    datetime_fields = set(['created_datetime'])
+    results_key = 'data'
+    url = '/api/phone/voice-call-events'
+
+    def sync(self, state, config):
+        sync_thru, max_synced_thru = self.get_sync_thru_dates(state)
+        # API does not accept query parameters as of July 24, 2024.
+        # Check https://developers.gorgias.com/reference/list-voice-call-events for updates
+        query_params = {}
+        LOGGER.info(f'Starting fetch for {self.name} between {sync_thru} and {self.utcnow_iso}')
+        for row in self.cursor_get(self.url, query_params):
+            event = {k: self.transform_value(k, v) for (k, v) in row.items()}
+            curr_synced_thru: str = event[self.replication_key]
+            max_synced_thru = max(curr_synced_thru, max_synced_thru)
+            yield (self.stream, event)
+        self.update_bookmark(state, max_synced_thru)
+
+
+class VoiceCallRecordings(CursorStream):
+    name = 'voice_call_recordings'
+    replication_method = 'INCREMENTAL'
+    key_properties = ['id']
+    replication_key = 'created_datetime'
+    datetime_fields = set(['created_datetime'])
+    results_key = 'data'
+    url = '/api/phone/voice-call-recordings'
```

Comment:
this is ordered in descending, meaning we can take advantage of the config/bookmark date to stop fetching there's a record time stamp older than the sync date.